core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -280,6 +281,7 @@ object KafkaConfig {
   val MaxConnectionsPerIpProp = "max.connections.per.ip"
   val MaxConnectionsPerIpOverridesProp = "max.connections.per.ip.overrides"
   val ConnectionsMaxIdleMsProp = "connections.max.idle.ms"
+  val FailedAuthenticationDelayMsProp = "connection.failed.authentication.delay.ms"
```

Comment:
We perhaps want to validate that `connection.failed.authentication.delay.ms` is less than `connections.max.idle.ms`.